tests_with_redis.sh

```diff
@@ -19,7 +28,10 @@ then
   pytest -m 'dev'
 elif [ "$1" == "integrationtest" ]
 then
-  pytest -m 'not unittest'
+  pytest -m 'not unittest and not noauth'
+elif [ "$1" == "noauth" ]
```

Comment:
Would it make sense to also add this to github test workflow?

Reply:
I wanted to run it after the integrationtest. I only forgot to add it.